src/main/java/uk/gov/mca/beacons/service/BeaconsRegistrationService.java

```diff
@@ -1,12 +1,38 @@
 package uk.gov.mca.beacons.service;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import uk.gov.mca.beacons.service.beacon.Beacon;
+import uk.gov.mca.beacons.service.beacon.BeaconRepository;
 
 @SpringBootApplication
 public class BeaconsRegistrationService {
 
+  private static final Logger log = LoggerFactory.getLogger(
+    BeaconsRegistrationService.class
+  );
+
   public static void main(String[] args) {
     SpringApplication.run(BeaconsRegistrationService.class, args);
   }
+
+  @Bean
+  public CommandLineRunner demo(BeaconRepository repository) {
+    return args -> {
+      repository.save(new Beacon());
+      repository.save(new Beacon());
+      repository.save(new Beacon());
+
+      log.info("Beacons found with findAll");
+      log.info("--------------------------------");
+      for (Beacon beacon : repository.findAll()) {
+        log.info(beacon.toString());
```

Comment:
This bit was just for me to see if things were working, will definitely be removed!
![image](https://user-images.githubusercontent.com/32230328/106900591-98fe8a00-66ee-11eb-8c1a-d5b00b5ead52.png)